test_azfilesfuse.py

```diff
@@ -345,6 +344,34 @@ def test_write(self):
         # TODO: verify fails if file handle isn't open for write.  (RESPECT
         # ATTRIBUTES OF OPEN, CURRENTLY WE DON'T
     
+    # For this test, set "quota" to be the size of the quota you've created.
+    def test_quota(self):
+        quota = 1250000000
```

Comment:
The portal does quota by GB. it would be good here to state what this is in GB, not bytes, if possible.

Also, I think you can call the storage libs to get the quota. That way this could be dynamic based on the storage configured for the tests. I have mine set to 1gb, which won't match this, which I think is 1.25GB?

Reply:
I looked into it and fileservice.get_share_properties seems to only work with prim/secondary key. Since the tests store SAS this may not be possible. Though it is possible using a broader sas token would work. The tests could be modified to provide the primary or secondary key though and to even provision the storage.

Reply:
The 125 was my being stupid and typing "-bit instead of -byte" when sanity checking I got my # of 0's correct.

I'll try doing it with a SAS token, but providing the primary and secondary keys may be overkill given that nothing else in the lib requires it?  As with the other comment, I'll conform with whatever direction you wanted to take this.

Reply:
It may be overkill. I guess my comments were a longwinded way of saying so :)

I think it would be reasonable to make it a requirement that quota be set to 1GB on the share  under test. If that is the case, it would be good to add that to the docs though.